ando/engine.py

```diff
@@ -172,6 +172,36 @@ def is_AnDO(directory, verbose, webcall):
             return 0,None;
         else : return found_err
 
+def next_is_AnDO(path):
+    """
+
+    Check if file path adhere to AnDO.
+    Main method of the validator. uses other class methods for checking
+    different aspects of the directory path.
+
+
+    Args:
+        directory ([str]): [names of the directory to check]
+
+    Returns:
+        [bool]: [does the directory adhere to the ando specification]
+    """
+    paths=[]
+    conditions=[]
+    if type(path) == str:
+        for r, d, f in os.walk(path):
+            for file in f:
+                paths.append(os.path.join(r, file))
+    else:
+        paths = path
+    conditions.append(is_top_level(paths))
+    conditions.append(is_file_level(paths))
+    conditions.append(is_session(paths))
+    conditions.append(is_subject(paths))
+
+    return(any(conditions))
```

Comment:
Does this `any` here mean that having only the subject level present in a structure will make the test succeed? Don't think this is intended here, right?